src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -895,82 +893,57 @@ private static boolean doesExtendFXApplication(Class<?> mainClass) {
      */
     private static final int MAIN_WITHOUT_ARGS = 1;
     private static final int MAIN_NONSTATIC = 2;
-    private static int mainType = 0;
 
     /*
      * Return type so that launcher invokes the correct main
      */
```

Comment:
If you are doing anymore edits then I think this method description could be improved to say that it returns the main method type.

Reply:
Another suggestion, if you are editing more, is the description on LoadMainClass (in java.c) is confusingly worded, he currently says "Loads a class and verifies that the main class is present". It invokes the helper's checkAndLoadMain so maybe the comment could be adjusted to say that it calls checkAndLoadMain to load the main class.

Reply:
Updated